docs/ai-core-APIs.md

```diff
@@ -0,0 +1,195 @@
+# Available APIs
+
+last updated: 24.07.2024
```

Comment:
[minor] Technically not needed since this file is in git. Maybe helpful to leave in for better visibility, but might also not be updated when the list is updated 😅🤷🏻‍♂️